AUTHORS.md

```diff
@@ -1,12 +1,12 @@
 # Project Authors
 
-This project was initially created by [Leonardo Uieda](http://www.leouieda.com/).
-
-The following people have made contributions to the project (in alphabetical order by
-last name) and are considered "The Pooch Developers":
+The following people have made contributions to the project (in alphabetical
+order by last name) and are considered "The Pooch Developers":
 
 * [Anderson Banihirwe](https://github.com/andersy005)
-* [Hugo van Kemenade](https://github.com/hugovk)
+* [Hugo van Kemenade](https://github.com/hugovk) - Independent (Non-affiliated) (ORCID: [0000-0001-5715-8632](https://www.orcid.org/0000-0001-5715-8632))
+* [Kacper Kowalik](https://github.com/Xarthisius) - National Center for Supercomputing Applications, Univeristy of Illinois at Urbana-Champaign, USA (ORCID: [0000-0003-1709-3744](https://www.orcid.org/0000-0003-1709-3744))
 * [John Leeman](https://github.com/jrleeman)
-* [Santiago R. Soler](https://github.com/santisoler)
-* [Leonardo Uieda](http://www.leouieda.com/)
+* [Rémi Rampin](https://github.com/remram44) - New York University, USA (ORCID: [0000-0002-0524-2282](https://www.orcid.org/0000-0002-0524-2282))
+* [Santiago Soler](https://github.com/santisoler) - Universidad Nacional de San Juan, Argentina (ORCID: [0000-0001-9202-5317](https://www.orcid.org/0000-0001-9202-5317))
```

Comment:
```suggestion
* [Santiago Soler](https://github.com/santisoler) - CONICET, Argentina; Instituto Geofísico Sismológico Volponi, Universidad Nacional de San Juan, Argentina (ORCID: [0000-0001-9202-5317](https://www.orcid.org/0000-0001-9202-5317))
```

Reply:
Sorry, I always forget that one. Could you add it to the other projects as well?

Reply:
Yes, no problem!